lib/cylc/rundb.py

```diff
@@ -173,9 +173,69 @@ class CylcSuiteDAO(object):
     MAX_TRIES = 100
     TABLE_BROADCASTS = "broadcasts"
     TABLE_TASK_JOBS = "task_jobs"
+    TABLE_TASK_JOB_LOGS = "task_job_logs"
```

Comment:
Is this new table going to pose a problem for restarts on older suites, or will it be created automatically if missing?